buildenv/docker/mkdocker.sh

```diff
@@ -459,7 +460,9 @@ if [ $arch = x86_64 ] ; then
   echo " && $wget_O gettext.tar.gz http://ftp.gnu.org/gnu/gettext/gettext-$gettext_version.tar.gz \\"
   echo " && tar -xzf gettext.tar.gz \\"
   echo " && cd gettext-$gettext_version \\"
+if [ $dist$version != centos6 ] ; then
```

Comment:
This is in a context that only applies for `$dist = centos`; the test can be simplified to `$version != 6`.

Reply:
Update made.